frontend/Dockerfile

```diff
@@ -20,7 +21,10 @@ RUN wget http://archive.apache.org/dist/hadoop/core/hadoop-$HADOOP_VERSION/hadoo
 
 # Dr Elephant Instalation
 RUN git clone https://github.com/linkedin/dr-elephant.git dr-elephant-sources
-RUN cd /dr-elephant-sources && ./compile.sh &&  cd dist && unzip dr*.zip  && rm *.zip && mv $PWD/dr*/ /dr-elephant
+RUN sed -i "s/hadoop_version=.*/hadoop_version=$HADOOP_VERSION/g" /dr-elephant-sources/app-conf/compile.conf
+RUN sed -i "s/spark_version=.*/spark_version=$SPARK_VERSION/g" /dr-elephant-sources/app-conf/compile.conf
+RUN cat /dr-elephant-sources/app-conf/compile.conf
```

Comment:
Step 14 : RUN sed -i "s/hadoop_version=.*/hadoop_version=$HADOOP_VERSION/g" /dr-elephant-sources/app-conf/compile.conf
 ---> Running in 361377111d6b
sed: can't read /dr-elephant-sources/app-conf/compile.conf: No such file or directory